ironweb.d.ts

```diff
@@ -183,6 +188,11 @@ export class SDKError extends Error {
     rawError: Error;
 }
 
+export interface UserOrGroup {
+    type: "user" | "group";
+    id: string;
+}
```

Comment:
Also pretty minor, but it might be nice for future us when reading this file if you moved this up above where it's used.